### PR TITLE
Fixer likes to break stuff

### PR DESCRIPTION
**Pull request by AlexandruGG**
This PR:

* [x] breaks stuff

### CODE REVIEW

File: src/Collection.php [hunk 1]
```diff
@@ -864,7 +864,7 @@ public function span(callable ...$callbacks): CollectionInterface
              */
             static function (array $spanResult): CollectionInterface {
                 /**
-                 * @var Closure(Iterator<TKey, T>): Generator<TKey, T> $callback
+                 * @var Closure(Iterator<TKey, <|T>): Generator|TKey, T> $callback
```
Comment (AlexandruGG) [inline src/Collection.php:867]: broken 😞

File: src/Operation/Associate.php [hunk 2]
```diff
@@ -43,7 +43,7 @@ public function __invoke(): Closure
                 /**
                  * @param callable(T=, TKey=, Iterator<TKey, T>=): NewT $callbackForValues
                  *
-                 * @return Closure(Iterator<TKey, T>): Generator<NewTKey, NewT>
+                 * @return Closure(Iterator<TKey, <|NewTKey, NewT|T>): Generator>
```
Comment (AlexandruGG) [inline src/Operation/Associate.php:46]: broken 2

**Comment by AlexandruGG** [comment]
@drupol I just realised you posted this: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5921#issuecomment-970447967

Looks like the latest releases are breaking more stuff than before

**Comment by drupol** [comment]
I updated https://github.com/drupol/phpcsfixer-configs-php and it should work now.

**Comment by drupol** [comment]
I guess we can close this?

**Pull request closed without merging**